lib/auth/clt.go

```diff
@@ -406,17 +406,17 @@ func (c *Client) GetDomainName() (string, error) {
 	return domain, nil
 }
 
-// GetClusterCACert returns the CAs for the local cluster without signing keys.
-func (c *Client) GetClusterCACert() (*LocalCAResponse, error) {
```

Comment:
I think we might need to keep this method for compatibility with older auth servers (see my comment somewhere below).